notebooks/examples/CEMS.ipynb

```diff
@@ -0,0 +1,1130 @@
+{
```

Comment:
<p>Hmm yeah, these shapefiles are a little janky aren't they, since they include coastal waters not just the land boundaries. Let's use <code>contextily</code>  to get a nice basemap to display this info.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-cooperative/pudl/pull/823/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Reply:
Docs here: https://geopandas.org/gallery/plotting_basemap_background.html